internal/engine/templatefuncs.go

```diff
@@ -81,6 +83,20 @@ func unmarkdown(s *string) string {
 	return withoutLinebreaks
 }
 
+// truncateText truncate text to avoid overly long text on overview pages
+func truncateText(s *string, limit int) *string {
+	if s == nil {
+		return s
+	}
+	if len(*s) > limit {
+		// truncate at last space or newline before given character limit (+3 puts ellipsis *after* the last word)
+		cutoff := strings.LastIndexAny((*s)[:limit], " \n") + 3
+		t := truncate.Truncate(*s, cutoff, "...", truncate.PositionEnd)
```

Comment:
Is dit een library waard? https://github.com/aquilax/truncate/blob/master/truncate.go lijkt niet heel complex.

Reply:
Nu je het zegt, nee. Het kan ook veel simpeler.